.github/workflows/staging-build-docker.yml

```diff
@@ -464,13 +464,13 @@ jobs:
       - name: Set up AWS Cred
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
       - name: Retrieve plugin tags 
         run: |
           echo "p_tag_sql_kibana=$(.github/scripts/plugin_tag.sh opendistro-for-elasticsearch/sql)" >> $GITHUB_ENV
```

Comment:
Replace `.github/scripts/plugin_tag.sh` to new folder structure too

Reply:
globally replaced now.